src/WebUI/Services/ServiceDetails.scss

```diff
@@ -7,7 +7,6 @@
 
         &.bolt-table-cell {
             padding-left: 0px;
```

Comment:
we should avoid overriding bolt css. if at all needed, we should keep all of them in one place, so in future we can take care of it, if we can not have a css class on our own.